landingzones/tests/test_views.py

```diff
@@ -173,6 +173,40 @@ def test_render(self):
         self.assertIsNotNone(form.fields['configuration'])
 
 
+class TestLandingZoneUpdateView(TestViewsBase):
+    """Tests for the landing zone update view"""
+
+    def test_render(self):
+        """Test rendering of the landing zone update view"""
+        with self.login(self.user):
+            response = self.client.get(
+                reverse(
+                    'landingzones:update',
+                    kwargs={'landingzone': self.landing_zone.sodar_uuid},
+                )
+            )
+        self.assertEqual(response.status_code, 200)
+        # Assert form
+        form = response.context['form']
+        self.assertIsNotNone(form)
+        self.assertIsNotNone(form.fields['assay'])
+        self.assertIsNotNone(form.fields['description'])
```

Comment:
Make sure to also assert the expected fields are hidden with the `HiddenInput` widget.